composer.json

```diff
@@ -39,7 +39,8 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.3",
+		"php": ">=5.4",
+		"phpbb/phpbb": ">=3.2.0@dev",
```

Comment:
wont this install phpbb into the extensions vendor folder, when running composer for the ext?